File: Status_25_07_2022/pattern11.py
#!/usr/bin/python3

def pattern(row):
    i,j,k = 0,0,0
    while i < row:
        j = i+1
        k = i
        while j:
            print(k," ",end = '')
            k = -k
            j -= 1
        print()
        i += 1
        
        
pattern(int(input("Enter the number of rows: ")))

File: Status_25_07_2022/pattern11.c
#include <stdio.h>

void pattern(int row)
{
    int i = 0, j = 0, k = 0;
    while (i <= row)
    {
        j = i + 1;
        k = i;
        while (j)
        {
            printf("%d ", k);
            k = -k;
            j--;
        }
        printf("\n");
        i++;
    }
}

int main()
{
    int num = 0;
    printf("Enter the number of rows: ");
    scanf("%d", &num);
    pattern(num);
}